README.md

```diff
@@ -158,7 +158,7 @@ The list below contains the functionality that contributors are planning to deve
   * [x] [Hive (community plugin)](https://github.com/baineng/feast-hive)
   * [x] [Postgres (community plugin)](https://github.com/nossrannug/feast-postgres)
   * [x] [Trino (community plugin)](https://github.com/Shopify/feast-trino)
```

Comment:
Update this link to the in-repo version?

Reply:
Going to leave as is until we have documentation for the Trino plugin